docs/source/api/user-guide/combinators/non-adaptive-composition.rst

```diff
@@ -0,0 +1,136 @@
+(Non-Adaptive) Composition
+--------------------------
+
+Any functions that have not completed the proof-writing and vetting
+process may still be accessed if you opt-in to “contrib”. Please contact
+us if you are interested in proof-writing. Thank you!
+
+.. tab-set::
+
+    .. tab-item:: Python
+        :sync: python
+
+        .. code:: python
+
+            >>> import opendp.prelude as dp
+            >>> dp.enable_features("contrib")
```

Comment:
If we carry state over from prior pages, can we drop this boilerplate?

Reply:
I've filed:
- #2468 

We could have a single file which is referenced from each of these RSTs, but I kind of like being able to run each page from the top. We can come back to this.